codebasin/finder.py

```diff
@@ -159,11 +161,31 @@ def find(
                         f"{filename} found in definition of platform {p} "
                         + "but missing from codebase",
                     )
-            state.insert_file(e["file"])
+            filenames.add(e["file"])
+    for f in tqdm(
+        filenames,
+        desc="Parsing",
+        unit="file",
```

Comment:
IIRC I think you actually want a space behind file (i.e. `" file"), otherwise it connects with the text prior to it

Reply:
Fixed in 4703339b54637b119aaba86ac73b37537aba4505.